src/main/java/core/basesyntax/service/impl/CsvReaderImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.CsvReader;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+
+public class CsvReaderImpl implements CsvReader {
+    private static final int SKIP_LINES = 1;
+
+    @Override
+    public List<String> readLines(String fileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            return reader.lines()
+                    .skip(SKIP_LINES)
+                    .toList();
```

Comment:
According to the checklist, you should avoid hardcoding values in your solution. It's better to pass the number of lines to skip as a parameter.